src/random/distributions/BinaryDistribution.hh

```diff
@@ -0,0 +1,64 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BinaryDistribution.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Macros.hh"
+#include "base/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Select one of two options with a given probability.
+ *
+ * The constructor argument is the chance of returning `true`, with an optional
+ * second argument for normalizing with a fraction of `false` values.
+ * \code
+    BinaryDistribution snake_eyes(1, 35);
+    if (snake_eyes(rng))
+    {
+        // ...
+    }
+    BinaryDistribution also_snake_eyes(1.0 / 36.0);
+   \endcode
+ *
+ * \note Since `true` is 1 and `false` is 0, the constructor argument is the
+ * _opposite_ order of std::discrete_distribution.
+ */
+class BinaryDistribution
```

Comment:
Thoughts on naming this `BernoulliDistribution`?

Reply:
Wow I'm a dum-dum, I didn't even know of that distribution's existence (or that a binary distribution is called a Bernoulli distribution). That's exactly what this should be. I'll update. Thanks!